libcloudforensics/gcp.py

```diff
@@ -673,16 +672,16 @@ def form_operation(self, operation_name):
 
     Raises RuntimeError:
       If resource type is not defined as a type which extends the
-      GoogleGomputeBaseResource class.
+      GoogleComputeBaseResource class.
     """
     resource_type = self.get_resource_type()
     module = None
     if resource_type not in ['compute#instance', 'compute#snapshot',
                              'compute#disk']:
       raise RuntimeError((
-          'Compute resource Type {0:s} is not one of the defined types in '
-          'libcloudforensics library (Instance, Disk or Snapshot) '
-      ).format(resource_type))
+                           'Compute resource Type {0:s} is not one of the defined types in '
+                           'libcloudforensics library (Instance, Disk or Snapshot) '
+                         ).format(resource_type))
```

Comment:
Consider building this string in a `msg =` variable before for better readability

Reply:
Addressed, and I changed all other similar cases for harmonization.